testing/regress/ecl/spray_test.ecl

```diff
@@ -24,25 +24,119 @@ import Std.File AS FileServices;
 //noThorLCR
 //noThor
 
-string OriginalTextFilesIp := '.' : STORED('OriginalTextFilesIp');
-string OriginalTextFilesEclPath := '' : STORED('OriginalTextFilesEclPath');
-string OriginalTextFilesOsPath := '' : STORED('OriginalTextFilesOsPath');
-
-File := OriginalTextFilesOsPath + '/download/spray_test.txt';
-RecordSize := 11;
-ClusterName := 'mythor';
-DestFile := '~regress::spray_test.txt';
-Timeout := -1;
-ESPportIP := 'http://127.0.0.1:8010/FileSpray';
-
-FileServices.SprayFixed(OriginalTextFilesIp,
-                        File,
-                        RecordSize,
-                        ClusterName,
-                        DestFile,
-                        Timeout,
-                        ESPportIP,
-                        ALLOWOVERWRITE := true);
-
-ds := DATASET(DestFile, { string name, unsigned age, boolean good }, csv);
-output(ds);
+//version sprayFixed=true
+//version sprayFixed=false
+
+import ^ as root;
+
+boolean sprayFixed := #IFDEFINED(root.sprayFixed, true);
+
+unsigned VERBOSE := 0;
+
+Layout_Person := RECORD
+  STRING3  name;
+  UNSIGNED2 age;
+  BOOLEAN good;
```

Comment:
I was concerned that this would generate true/false in the output, but that doesn't seem to be the case.